docs/source/models/gbd2017/causes/lower_respiratory_infections/index.rst

```diff
@@ -46,10 +49,104 @@ Data Description
 
    * - State
      - Definition
+   * - S
+     - Susceptible but does not currently have LRI
    * - I
      - Currently infected and having the condition
+
+.. list-table:: **States Data**
+   :widths: 20 25 30 30
+   :header-rows: 1
+
+   * - State
+     - Measure
+     - Value
+     - Notes
    * - S
-     - Susceptible but does not currently have LRI
+     - prevalence
+     - 1-prevalence_c322
+     -
+   * - S
+     - excess mortality |br| rate
+     - 0
+     -
+   * - S
+     - disabilty weights
+     - 0
+     -
+   * - I
+     - prevalence
+     - prevalence_c322
+     -
+   * - I
+     - excess mortality |br| rate
+     - :math:`\frac{\text{deaths_c322}}{\text{population * prevalence_c322}}`
+     -
+   * - I
+     - disability weights
+     - disability_weight_s670* |br| prevalence_s670+ |br| disability_weight_s669* |br| prevalence_s669
```

Comment:
Maybe:

:math:`(\text{disability_weight_s670}\times\text{prevalence_s670})(\text{disability_weight_s669}\times\text{prevalence_s669})`

or without the parentheses--i think anything that gets rid of the |br|s works, unless perhaps my locally compiled version isn't what it will look like once published, and those are line breaks or something?

Reply:
S

o






r




r


y

Reply:
Sorry for the confusion, I fixed the |br| in my second commit. It was not right before